lib/ex_doc/formatter/html/templates/api_reference_template.eex

```diff
@@ -4,7 +4,7 @@
 
 <%= if nodes_map.modules != [] do %>
   <section class="details-list">
-    <h2 id="modules" class="section-heading">Modules</h2>
```

Comment:
REVIEWER: Not needed, this is injected by `ExDoc.Formatter.HTML.Templates.link_heading/5`